src/api/app/models/kiwi/image.rb

```diff
@@ -141,7 +141,7 @@ def outdated?
   end
 
   def default_package_group
-    package_groups.find_or_create_by(kiwi_type: :image)
```

Comment:
It's not causing any problem that don't create this if it's missing?

Reply:
uhhmm, good point!